packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -21,26 +21,25 @@ contract OptimismPortal is Initializable, ResourceMetering {
      */
     uint8 public constant VERSION = 1;
 
+    /**
+     * @notice Version of the deposit event.
+     */
+    uint256 internal constant DEPOSIT_VERSION = 1;
+
     /**
      * @notice Emitted when a transaction is deposited from L1 to L2. The parameters of this event
      *         are read by the rollup node and used to derive deposit transactions on L2.
      *
      * @param from       Address that triggered the deposit transaction.
      * @param to         Address that the deposit transaction is directed to.
-     * @param mint       Amount of ETH to mint to the sender on L2.
-     * @param value      Amount of ETH to send to the recipient.
-     * @param gasLimit   Minimum gas limit that the message can be executed with.
-     * @param isCreation Whether the message is a contract creation.
-     * @param data       Data to attach to the message and call the recipient with.
+     * @param version    Version of this deposit transaction event.
+     * @param opaqueData ABI encoded deposit data to be parsed off-chain.
      */
     event TransactionDeposited(
         address indexed from,
         address indexed to,
-        uint256 mint,
-        uint256 value,
-        uint64 gasLimit,
-        bool isCreation,
-        bytes data
+        uint256 indexed version,
```

Comment:
Making this indexed saves about 100 gas. I'm not attached to it tho.

Reply:
is it the same with opaqueData? Sounds more useful to that indexed instead.

Reply:
Indexed arguments are taken from the stack, so if `opaqueData` were indexed then it would be hashed to fit into 256 bits.